modules/ui/CaptionUI.py

```diff
@@ -318,7 +328,9 @@ def draw_mask(self, event):
         color = None
 
         if event.state & 0x0100 or event.num == 1:  # left mouse button
-            color = (255, 255, 255)
+            alpha = float(self.enable_mask_editing_alpha_var.get())
```

Comment:
This breaks if the user writes something other than a number into the input fields. Some kind of validation would be nice.

Reply:
Done